.github/workflows/validate.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       # Otherwise we would not know if the problem is tied to the Node.js version
       fail-fast: false
       matrix:
-        node: [12, 14, 16]
+        # TODO: relax `'16.9.1'` to `16` once GitHub has 16.9.1 cached. 16.9.0 is broken due to https://github.com/nodejs/node/issues/40030
+        node: [12, 14, '16.9.1']
```

Comment:
Can't use `'^16.9.1'` because codecov flags do not support carets:

> Flags Must consist only of alphanumeric characters, '_', '-', or '.' and not exceed 45 characters
-- https://docs.codecov.com/docs/flags